host-configs/CVX/cvx-base.cmake

```diff
@@ -0,0 +1,41 @@
+set(ENABLE_GTEST_DEATH_TESTS ON CACHE BOOL "" FORCE)
+set(ENABLE_CALIPER ON CACHE BOOL "")
+
+set(ENABLE_HYPRE ON CACHE BOOL "" FORCE)
+# If not defined as argument, take from the environment...
+if(NOT DEFINED ENABLE_HYPRE)
+  set(ENABLE_HYPRE "$ENV{ENABLE_HYPRE}" CACHE BOOL "" FORCE)
+endif()
+# ... and then check the value.
+if(ENABLE_HYPRE)
+  set(GEOSX_LA_INTERFACE "Hypre" CACHE STRING "" FORCE)
+else()
+  set(ENABLE_HYPRE OFF CACHE BOOL "" FORCE)
+endif()
+
+# Same pattern
+if(NOT DEFINED ENABLE_TRILINOS)
+  set(ENABLE_TRILINOS "$ENV{ENABLE_TRILINOS}" CACHE BOOL "" FORCE)
+endif()
+if(ENABLE_TRILINOS)
+  set(GEOSX_LA_INTERFACE "Trilinos" CACHE STRING "" FORCE)
+else()
+  set(ENABLE_TRILINOS FALSE CACHE BOOL "" FORCE)
+endif()

```

Comment:
I you can, do not rely on the `env` variables.
They are used in the very specific context of the `CI`, where it makes sense.
But you'd better not use it if you can.

Reply:
I think those are some leftovers from experiments.
Removed "env".